### PR TITLE
Fixing most of the issues raise by pylint

### PR DESCRIPTION
**Pull request by cashew22**

### CODE REVIEW

**Comment by marcolivierarsenault** [comment]
Can you add isort and flake8 to both CI workflows

**Comment by marcolivierarsenault** [comment]
ex:
https://github.com/marcolivierarsenault/tefnut/blob/main/.github/workflows/python-app.yml#L38
https://github.com/marcolivierarsenault/tefnut/blob/main/.github/workflows/python-app.yml#L50

On pourrait regarder pour Black itou

**Comment by cashew22** [comment]
J'ai ajouté isort et flake8, black est déja la.

File: .github/workflows/pull.yml [hunk 1]
```diff
@@ -25,10 +25,13 @@ jobs:
     steps:
         - uses: "actions/checkout@v2"
         - uses: "actions/setup-python@v1"
+        - uses: isort/isort-action@v1.0.0
```
Comment (marcolivierarsenault) [inline .github/workflows/pull.yml:28]:
```suggestion
        - uses: isort/isort-action@v1.0.0
        - uses: jpetrucciani/black-check@master
```

File: .github/workflows/pull.yml [hunk 2]
```diff
@@ -25,10 +25,13 @@ jobs:
...
           with:
             python-version: "3.x"
-        - run: python3 -m pip install black
+        - run: python3 -m pip install black flake8
         - run: black .
```
Comment (marcolivierarsenault) [inline .github/workflows/pull.yml:32]:
```suggestion
```
Delete this one

**Review by marcolivierarsenault — COMMENTED** [review]
- Should use the black check instead of the formater. 
- Can you move this logic to the `push.yml` flow too
Other than that LGTM

**Pull request merged**